src/assignment/views/AssignmentOverview.tsx

```diff
@@ -545,7 +539,7 @@ const AssignmentOverview: FunctionComponent<AssignmentOverviewProps> = ({
 				return renderActions(assignment);
 
 			default:
-				return cellData;
+				return '-';
```

Comment:
Het is nuttiger om de gebruiker niet-geformateerde of worst-case `[object Object]` te tonen, zowel voor bug hunting als user-value

Reply:
klopt, maar cellData crashed aangezien het geen reactNode is
ik zal het vervangen door JSON.stringify